rakelib/support/sessions_serializer.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require './rakelib/support/cookies.rb'
+
+class SessionsSerializer
+  def initialize(_arg)
+    @sessions = []
+    @redis = Redis.current
+  end
+
+  def load
+    raise 'subclass must implement #load'
```

Comment:
Is `load` part of a pattern that we're just not utilizing here? Neither of the subclasses implement nor use it from what I can see

Reply:
good catch I moved that logic into each of the sub classes initialize method, will remove